tools/tertiary-analysis/scanpy/scanpy-plot-trajectory.xml

```diff
@@ -50,6 +50,10 @@ PYTHONIOENCODING=utf-8 scanpy-cli plot paga
       <option value="fr" selected="true">Fruchterman-Reingold</option>
       <option value="rt">Reingold-Tilford</option>
     </param>
+    <param name="legend_loc" argument="--legend-loc" type="select" label="Location of legend">
```

Comment:
You also need to add a section within the command part above that uses the value of this new param, like what is done with `--layout` on line 14, but for `--legend-loc`. Thanks!

Reply:
I don't understand. This doesn't look like this: https://galaxyproject.org/admin/tools/adding-tools/
It is just
```
#end if
    --layout-loc ${layout-loc}
```

Reply:
Yes, if `legend_loc` is not optional (since it is a drop down that has a default), then you just need to add this without any if as you suggest.

Reply:
OK! Woop wood!